### ISSUE TITLE
Lora fine-tuning training example code?

### ISSUE THREAD

**Issue by kabachuha** [character-ai/Ovi#70]
Hello, thank you for the amazing project!

It is a great base model capable both of video and audio stuff. However, it would be even greater if there was an easy way to insert new objects, concepts/vfx, sound/sfx, and voices.

Could you provide an example train code for LoRA? It's very interesting which loss functions, and in which ratios, select for the video and audio parts. Can we train only on videos / audios by masking loss? How difficult is it to train on a new speaker identity?

As another commenter said here, if you give an example code, it can be adopted by different known trainers, such as AI Toolkit and Diffusion Pipe, so the community can do entirely new things with it.

Thank you for the work again
